### PR TITLE
Remove BlockWorker.moveBlock APIs

### PR DESCRIPTION
**Pull request by rongrong**
The moveBlock APIs are very light wrappers on top of the API for LocalBlockStore. They don't really need to exist.

### CODE REVIEW

File: tests/src/test/java/alluxio/client/hadoop/FileSystemRenameIntegrationTest.java [hunk 1]
```diff
@@ -237,7 +237,7 @@ public void basicRenameTest7() throws Exception {
     // Due to Hadoop 1 support we stick with the deprecated version. If we drop support for it
     // FSDataOutputStream.hflush will be the new one.
     //#ifdef HADOOP1
-    o.sync();
+//    o.sync();
```
Comment (beinan) [inline tests/src/test/java/alluxio/client/hadoop/FileSystemRenameIntegrationTest.java:240]: uncomment?

File: core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java [hunk 2]
```diff
@@ -270,14 +271,13 @@ public Response getWebUIBlockInfo(@QueryParam("path") String requestPath,
...
-              // we are viewing at the context of this worker.
-              uiFileInfo.addBlock(blockMeta.getBlockLocation().tierAlias(), blockId, blockSize, -1);
-            }
+            Optional<BlockMeta> blockMeta = mBlockWorker.getLocalBlockStore()
+                .getVolatileBlockMeta(blockId);
+            // The block last access time is not available. Use -1 for now.
+            // It's not necessary to show location information here since
+            // we are viewing at the context of this worker.
+            blockMeta.ifPresent(meta -> uiFileInfo.addBlock(meta.getBlockLocation().tierAlias(),
+                blockId, meta.getBlockSize(), -1));
```
[the remark this answers is not in the document]
Comment (rongrong) [reply core/server/worker/src/main/java/alluxio/worker/AlluxioWorkerRestServiceHandler.java:280]: That's original code. So this is out of the scope of this PR. Reading the comment it seems that using -1 is also a workaround so it's probably better to keep it here.

File: core/server/worker/src/main/java/alluxio/worker/block/evictor/AbstractEvictor.java [hunk 3]
```diff
@@ -131,46 +125,39 @@ protected StorageDirEvictorView cascadingEvict(long bytesToBeAvailable,
...
-            candidateDirView.markBlockMoveOut(blockId, block.getBlockSize());
-            plan.toEvict().add(new Pair<>(blockId, candidateDirView.toBlockStoreLocation()));
-          }
-        } catch (BlockDoesNotExistException e) {
-          continue;
+        Optional<BlockMeta> block = mMetadataView.getBlockMeta(blockId);
+        if (block.isPresent()) {
+          candidateDirView.markBlockMoveOut(blockId, block.get().getBlockSize());
+          plan.toEvict().add(new Pair<>(blockId, candidateDirView.toBlockStoreLocation()));
         }
```
[the remark this answers is not in the document]
Comment (rongrong) [reply core/server/worker/src/main/java/alluxio/worker/block/evictor/AbstractEvictor.java:132]: It would complain variable used in lambda should be final.

**Review by beinan — APPROVED** [review]

**Comment by beinan** [comment]
alluxio-bot, merge this please.

**Pull request merged**